build.sh

```diff
@@ -364,7 +364,7 @@ fi
 if [ $__CrossBuild == 1 ]; then
     export CROSSCOMPILE=1
     if ! [[ -n "$ROOTFS_DIR" ]]; then
-        export ROOTFS_DIR="$__ProjectRoot/cross/rootfs/$__BuildArch"
+        export ROOTFS_DIR="$__ProjectRoot/cross/rootfs"
     fi
```

Comment:
I know it won't change my definition, I usually never do that as it could be confusing when you see this.